tests/conftest.py

```diff
@@ -311,7 +311,7 @@ class Category:
 @pytest.fixture(autouse=True)
 def no_usage_stats(monkeypatch):
     # Do not generate usage stats from test runs
-    monkeypatch.setenv("GE_USAGE_STATS", "False")
+    monkeypatch.delenv("GE_USAGE_STATS", raising=False)
```

Comment:
Honing-in on this change @alexsherstinsky. Is this an intended change to the fixture?

Reply:
@Shinnnyshinshin It was intended in order to fix a PyCharm warning.